scipy/special/_basic.py

```diff
@@ -3087,6 +3090,139 @@ def factorialk(n, k, exact=True):
     return _exact_factorialx_array(n, k=k)
 
 
+def stirling2(N, K, *, exact=True):
+    r"""Generate Stirling number(s) of the second kind.
+
+    Stirling numbers of the second kind count the number of ways to
+    partition a set with N elements into K non-empty subsets.
+
+    The values this function returns are calculated using a dynamic
+    program which avoids redundant computation across the subproblems
+    in the solution. For array-like input, this implementation also 
+    avoids redundant computation across the different Stirling number
+    calculations.
+
+    The numbers are sometimes denoted
+    .. math::
```

Comment:
```suggestion
    The numbers are sometimes denoted

    .. math::
```

I think this should fix the rendering issue. At least the blank line is used throughout other `special` docstrings.

Reply:
![Screen Shot 2023-07-19 at 7 09 29 PM](https://github.com/scipy/scipy/assets/5400493/165ff44f-f02d-4683-aafe-fef59fdd9d22)
Adding the newline fixed it, thanks @dschmitz89